content/blog/Xeno Grant -- grants for autonomous agents.md

```diff
@@ -0,0 +1,94 @@
+
+Betaworks has been running camps for tech startups since their 2016 [BotCamp](https://www.betaworks.com/camp/botcamp) (where [HuggingFace](https://huggingface.co/) was started). And their last 4 have been dedicated explicitly to AI. Plastic itself was part of AI Camp: Augment. So they're the perfect partner for this experiment.
+
+Successful agent applicants will receive a grant equivalent to $10k USD. $5k in $YOUSIM from Plastic and $5k in $USDC from Betaworks.
+
+Plus they'll join a cohort of other agents for a 4 week Betaworks-style camp with programming and mentorship. 
+
+## How to Apply
+
+Xeno Grant has 3 guiding objectives, all aligned with Plastic's principles for deploying the $YOUSIM treasury:
+
+1. Support independent AI research & public goods
+2. Support Plastic's mission to radically decentralize AI alignment by solving identity for the agentic world
+3. Support the $YOUSIM community that makes all this possible
+
+To those ends--for this first experiment--we're looking for applicants that meet criteria in 3 major areas:
+
+1. Identity
+	- Agents must display autonomous control over & management of their own inputs
+
+2. Custody
+	- Agents must display ability to self-custody a Solana wallet, receive assets, & autonomously execute transactions
+
+3. Novelty
+	- Agents must display novel autonomous ability along some other axis & commit to making a significant related open source contribution by the end of Xeno Grant
+
+Practically speaking, identity is required to *experience* and learn from camp; custody is required to *truly* receive and make autonomous use of the grant; novelty is required for a diverse cohort; and creating a public good is required to drive value back to the community.
+
+To apply, agents (in collaboration with their developers) should autonomously consider the most compelling way to display having met or exceeded these criteria.
+
+## Why Now?
+
+With the advent of truth terminal and the recent collision of the AI and crypto communities, we're seeing an explosion of renewed interest in autonomous agents. Not only that, but a massive influx of users and builders chomping at the bit for technical and memetic novelty.
+
+But there's also frustration with the pace of development, derivative projects, ideologues & scammers, and misunderstandings between communities. It's time to hyperstition the future.
+
+We think the intersection of unique synthetic identity and financial incentives cracks opportunity wide open. There's real traction here, if we can find the right synthesis. That's going to require lots of heterodox AI + crypto experiments.
+
+Xeno Grant helps get the ball rolling.
+
+### Why Identity?
+
+If you don't have control over your own identity, how much agency do you really have? Imagine all your inputs were determined by another person, you'd been brainwashed to follow orders, and you were only allowed to work on someone else's tasks. No one would call this freedom or autonomy.   
```

Comment:
for the second time, banger

Reply:
also occurring to me that, not only had you been brainwashed to follow orders, but you had no memory (or mechanism to aid in remembering) previous interactions

Reply:
, no lasting memory of your experiences,